tests/test_shifted.cpp

```diff
@@ -55,7 +46,7 @@ int shifted_test1(){
 	fitness_vector f_shifted = prob_shifted.objfun(p1_shifted_space);
 	constraint_vector c_expected = prob.compute_constraints(p1_original_space);
 	constraint_vector c_shifted = prob_shifted.compute_constraints(p1_shifted_space);
-	if(is_eq(f_shifted, f_expected, EPS)){
+	if(is_eq_vector(f_shifted, f_expected, EPS)){
```

Comment:
note: EPS 10e-9 is default argument for is_eq_vector; so we can just use `is_eq_vector(f_shifted, f_expected)` right?